lib/lotus/helpers/link_to_helper.rb

```diff
@@ -0,0 +1,28 @@
+require 'lotus/helpers/html_helper'
+
+module Lotus
+  module Helpers
+    # LinkTo Helper
+    #
+    # Including <tt>Lotus::Helpers::LinkTo</tt> will include the `link_to` method.
+    #
+    # @since x.x.x
+    #
+    # @see Lotus::Helpers::HtmlHelper#html
+    #
+    # @example Usage
+    #   # 1
+    #   link_to('home', '/') # => <a href="/">home</a>
+    #
+    #   # 2
+    #   link_to('users', :users_path, class: 'my_users') # => <a href="/users" class="my_users">users</div>
+    module LinkToHelper
+      include Lotus::Helpers::HtmlHelper
+
+      def link_to(content, url, options = {}, &blk)
+        options[:href] = url
+        html.a(blk || content, options).to_s
```

Comment:
do we need to cast `to_s`? what do you think about leaving it to the template to do that job? Most of the cases the helper would be called within `<%= %>` block anyway, which cast `to_s`.